dimens.xml

```diff
@@ -928,6 +990,8 @@
     <dimen name="restricted_icon_size">16.0dip</dimen>
     <dimen name="restricted_padlock_pading">4.0dip</dimen>
     <dimen name="ripple_radius">22.599976dip</dimen>
+    <dimen name="rounded_corner_content_padding">0.0dip</dimen>
+    <dimen name="rounded_corner_radius">0.0dip</dimen>

```

Comment:
Line 993 and 994 is used to make rounded corners and it is only for android 8.1.0 only.